examples/plot_calculate_classify_ndvi.py

```diff
@@ -0,0 +1,147 @@
+"""
+Calculate and Classify Normalized Difference Results with EarthPy
+==================================================================
+
+Learn how to calculate and classify normalized difference indices in Python using EarthPy.
+This example shows how to calculate and classify the normalized difference vegetation
+index (NDVI) using Landsat 8 data.
+
+"""
+
+###############################################################################
+# Calculating Normalized Difference in Python Using EarthPy
+# ---------------------------------------------------------
+#
+# .. note::
+#    The examples below will show you how to use the ``normalized_diff()`` function
+#    to calculate the normalized difference vegetation index (NDVI), a commonly
+#    used remotely sensed index for quantifying vegetation health.
+#
+# The example below walks you through a typical workflow for calculating the normalized
+# difference vegetation index (NDVI) using Landsat 8 data with EarthPy. NDVI provides
+# a measure of healthy vegetation and ranges in value from -1 to 1. Values closer to
+# 1 represent healthy, green vegetation. NDVI can be calculated from Landsat 8 data
+# using band 4 (red) and band 5 (near-infrared).
+#
+# First, you will create a stack of bands using Landsat 8 data and then calculate
+# NDVI using the ``normalized_diff()`` function. Then, you will plot the NDVI results
+# using a colorbar legend with continuous values. Last, you will classify the NDVI
+# results using threshold values and plot the classified data with a categorical legend.
+
+
+###############################################################################
+# Import Packages
+# ---------------
+#
+# To begin, import the needed packages. You will use a combination of several EarthPy
+# modules including spatial and plot.
+
+import os
+from glob import glob
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.colors import ListedColormap
+import earthpy as et
+import earthpy.spatial as es
+import earthpy.plot as ep
+
+# Get data and set your home working directory
+data = et.data.get_data("cold-springs-fire")
+
+
+###############################################################################
+# Import Example Data
+# -------------------
+#
+# To get started, make sure your directory is set. Then, create a stack from all of
+# the Landsat .tif files (one per band).
+
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+
+# Stack the Landsat 8 bands
+# This creates a numpy array with each "layer" representing a single band
+landsat_path = glob(
+    "data/cold-springs-fire/landsat_collect/LC080340322016072301T1-SC20180214145802/crop/*band*.tif"
+)
+landsat_path.sort()
+arr_st, meta = es.stack(landsat_path)
+
+
+###############################################################################
+# Calculate Normalized Difference Vegetation Index (NDVI)
+# -------------------------------------------------------
+
```

Comment:
@jlpalomino i just tried to edit this via a patch but GH is not allowing it right now. please add a # to this line and the lines below as commented to ensure the text renders as text rather than comments

Reply:
@lwasser Apologies for that! The #s have been added to these lines. The PR is ready for review.